cmd/restic/cmd_check_integration_test.go

```diff
@@ -34,3 +35,86 @@ func testRunCheckOutput(t testing.TB, gopts global.Options, checkUnused bool) (s
 	})
 	return buf.String(), err
 }
+
+func testRunCheckOutputWithOpts(t testing.TB, gopts global.Options, opts CheckOptions, args []string) (string, error) {
+	buf, err := withCaptureStdout(t, gopts, func(ctx context.Context, gopts global.Options) error {
+		gopts.Verbosity = 2
+		_, err := runCheck(context.TODO(), opts, gopts, args, gopts.Term)
+		return err
+	})
+	return buf.String(), err
+}
+
+func TestCheckFullOutput(t *testing.T) {
```

Comment:
Please merge the tree tests into a table test (see for example `TestPrepareCheckCache`; you bascially create a struct that contains the expected inputs and output and then use a loop to iterate over all of them). Almost all of their code is shared.